src/components/base/BaseDatepicker.vue

```diff
@@ -0,0 +1,49 @@
+<script setup lang="ts">
+
+withDefaults(
+  defineProps<{
+    modelValue?: string;
```

Comment:
I believe this prod is not used

Reply:
this component was replaced by custom datepicker, I have not cleaned up.